dramatiq_sqs/broker.py

```diff
@@ -84,9 +84,13 @@ def __init__(
         self.tags: Optional[Dict[str, str]] = tags
         self.sqs: Any = boto3.resource("sqs", **options)
 
+    @property
+    def consumer_class(self):
+        return SQSConsumer
```

Comment:
I would prefer the consumer class be a constructor argument.

Reply:
I chose this approach to be consistent with current implementations  of the [Redis broker](https://github.com/Bogdanp/dramatiq/blob/master/dramatiq/brokers/redis.py#L122) and [RabbitMQ broker](https://github.com/Bogdanp/dramatiq/blob/master/dramatiq/brokers/rabbitmq.py#L125).
Could we postpone implementation as a constructor argument until this is implemented in the mentioned brokers?